gen/__init__.py

```diff
@@ -669,6 +670,10 @@ def generate(
     argument_dict['expanded_config_full'] = format_expanded_config(expanded_config_full)
     argument_dict['expanded_config'] = format_expanded_config(expanded_config_scrubbed)
 
+    # Initialize CA and add arguments (exhibitor_ca_certificate and exhibitor_ca_certificate_path
+    # init_exhibitor_ca_bootstrap(argument_dict)
```

Comment:
This comment appears to be an earlier version of the next line. If so, we can remove it. If not, is something missing?